js/npm/restify/Error.hx

```diff
@@ -0,0 +1,431 @@
+package js.npm.restify;
+
+typedef ErrorOptions = {
+    ?message : Dynamic,
+    ?statusCode : Int
+}
+
+@:native('errors.HttpError')
+extern class HttpError extends js.Error implements js.extern.Extern<'restify'>
```

Comment:
I'm not sure about the `@:native('errors.HttpError')` either...
What is it supposed to do ?